docs/UserGuide.md

```diff
@@ -185,15 +191,19 @@ This action is irreversible and all deleted tasks cannot be retrieved.
 
 Finds the tasks that match the given keyword(s) and is due within the time range specified from Harmonia.
 
-Format: `find [n/NAME_KEYWORD]... [t/TAG_KEYWORD]... [start/START_DATE] [end/END_DATE]`
+Format: `find [n/NAME_KEYWORD]... [d/DESCRIPTION_KEYWORD]... [start/START_DATE] [end/END_DATE] [p/PRIORITY]... [t/TAG]...`
+
+* If a `NAME_KEYWORD` is supplied by you, Harmonia finds the tasks whose name contains the `NAME_KEYWORD` specified. If multiple `NAME_KEYWORD`s are supplied by you, Harmonia finds the tasks whose name contains any of the `NAME_KEYWORD`s supplied.
+
+* If a `DESCRIPTION_KEYWORD` is supplied by you, Harmonia finds the tasks whose description contains the `DESCRIPTION_KEYWORD` specified. If multiple `DESCRIPTION_KEYWORD`s are supplied by you, Harmonia finds the tasks whose description contains any of the `DESCRIPTION_KEYWORD`s supplied.
 
-* If only `NAME_KEYWORD` is used, Harmonia finds the tasks whose name contains any of the `NAME_KEYWORD` specified.
+* If both `START_DATE` and `END_DATE` are supplied by you, Harmonia finds the tasks whose deadline is between the `START_DATE` and `END_DATE` inclusive.
 
-* If only `TAG_KEYWORD` is used, Harmonia finds tasks whose tag contains any of the `TAG_KEYWORD` specified.
+* If only one of the `START_DATE` or `END_DATE` is supplied by you, Harmonia ignores the unspecified field. For example, if only `START_DATE` is supplied by you, Harmonia finds all the tasks with a deadline that is after and including `START_DATE`.
```

Comment:
Should this be "If only one of either `START_DATE` or `END_DATE` is supplied by you..."